lib/results_validators/persons_validator.rb

```diff
@@ -101,7 +101,7 @@ def run_validation(validator_data)
                                                name: p.name)
           end
 
-          PersonsValidator.dob_validations(p.dob, competition.id, p.name).each do |validation|
+          PersonsValidator.dob_validations(p.dob, { name: p.name }, competition.id).each do |validation|
```

Comment:
For my suggestion above, this could be changed to `PersonsValidator.dob_validations(p.dob, competition.id, name: p.name)`

Reply:
Done